src/components/drivers/photo-viewer-wrapper.jsx

```diff
@@ -26,15 +26,31 @@ export default class PhotoViewerWrapper extends Component {
   }
 
   componentDidMount() {
-    // spike on using promises and a different loader or adding three js loading manager
+    this.loadPhoto();
```

Comment:
All the photo loading code only ran here, on mount. I moved it to loadPhoto, which also runs if the filePath prop changes now.